includes/data-stores/class-wc-admin-reports-orders-stats-data-store.php

```diff
@@ -436,10 +436,12 @@ public static function update( $order ) {
 		);
 
 		if ( 'shop_order_refund' === $order->get_type() ) {
-			$parent_order        = wc_get_order( $order->get_parent_id() );
-			$data['customer_id'] = WC_Admin_Reports_Customers_Data_Store::get_or_create_customer_from_order( $parent_order );
-			$data['parent_id']   = $parent_order->get_id();
-			$format[]            = '%d';
+			$parent_order = wc_get_order( $order->get_parent_id() );
+			if ( is_a( $parent_order, 'WC_Order' ) ) {
```

Comment:
I don't think this section will be necessary after rebase.

Reply:
I think that it is - there's still a chance the parent order doesn't exist.